lib/screens/user_registration/registration.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:teesco/screens/user_registration/widgets/institute_name_input.dart';
+import 'package:teesco/screens/user_registration/widgets/phone_number_feild.dart';
+import 'package:teesco/screens/user_registration/widgets/signup_button.dart';
+import 'package:teesco/screens/user_registration/widgets/username_input.dart';
+import 'package:teesco/screens/user_registration/widgets/confirm_password_feild.dart';
+import '../login/widgets/email_field.dart';
+import '../login/widgets/password_field.dart';
+
+//TODO: Name feild Widget     W1
+//TODO: Email Feild Widget    W5
+//TODO: Name of Institute     W1
+//TODO: Phone Number          W2
+//TODO: Password              W3
+//TODO: Confirm Password      W3
+//TODO: Register Button       W4
+
+class Registration extends StatelessWidget {
+  TextEditingController EmailController;
```

Comment:
variables must be named using `lowerCaseCamel`
```suggestion
  TextEditingController emailController;
```

Reply:
Fixed